Dockerfile

```diff
@@ -200,6 +200,10 @@ RUN ./install-python.sh 3.13
 # Particularly important for Python virtual environments
 ENV REQUESTS_CA_BUNDLE=/etc/pki/tls/certs/ca-bundle.crt
 
+# Set paths for openssl/urllib
+ENV SSL_CERT_FILE=/etc/pki/tls/certs/ca-bundle.crt
+ENV SSL_CERT_DIR=/etc/pki/tls/certs
+
```

Comment:
One minor question, maybe we should put this at the end of the file, so it doesn't impact the build steps afterwards? Though the build passed and seems it doesn't impact the building.